core-referrals/core-referrals.gemspec

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require_relative 'lib/core/referrals/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'core-referrals'
+  spec.version = Core::Referrals::VERSION
+  spec.authors = ['Max Schridde']
+  spec.email = ['maxjschridde@gmail.com']
+
+  spec.summary = 'CORE extension for referrals backend'
+  spec.description = 'CORE::Referrals is a referrals extension for the CORE Platform.'
+  spec.post_install_message = 'CORE::Referrals installed'
+
+  spec.homepage = 'https://moonlight-labs.com/core/'
+  spec.required_ruby_version = '>= 3.1.0'
+
+  spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://moonlight-labs.com/core/'
+  spec.metadata['changelog_uri'] = 'https://moonlight-labs.com/core/core-referrals#changelog'
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  # spec.add_dependency 'activerecord', '~> 6.0'
+  spec.add_dependency 'base32h'
+  spec.add_dependency 'counter_culture'
```

Comment:
I'd be up for removing this dep in favor of vanilla counter cache.

Reply:
sounds good to me